### DISCUSSION TITLE
Is it possible to keep Result Panel open when you execute a query?

### DISCUSSION THREAD

**Discussion by xantari** [dbeaver/dbeaver#9137] [Q&A] [answered]
## System information: 
- Operating system (distribution) and version: Windows 10 64bit
- DBeaver version: 7.1.1

When executing a query, the result panel disappears, executes the query, then the result panel appears again. Kinda wierd everytime I run a query. As I recall the result panel would just stay open and a new tab would get displayed on it.

I don't recall previous versions doing this. Recently upgraded from 6.1.1 to 7.1.1.

**Comment by xantari** [comment]
So I tested 6.3.1 and the results tab stays open too. Using Alt+X shortcut key in 6.1.1 and 6.3.1 and it keeps the result panel open.

In 7.1.1 is closes the result panel, then executes the query and opens the result panel again, which is really strange from a user perspective.

Don't see a setting to lock the panel anywhere...

**Comment by serge-rider** [comment, marked as answer]
By default results panel is hidden during script execution (alt+x). When you execute a single query (ctrl+enter) it remains where it is.

You can change this behavior in preferences:
![image](https://user-images.githubusercontent.com/6398845/86297589-69414e00-bc04-11ea-89ba-4968056a4270.png)

**Comment by xantari** [comment]
That fixed it, thanks!!!
